Services/AccessControl/classes/class.ilAccess.php

```diff
@@ -783,7 +783,6 @@ function doStatusCheck($a_permission, $a_cmd, $a_ref_id,$a_user_id, $a_obj_id, $
 		$class = $objDefinition->getClassName($a_type);
 		$location = $objDefinition->getLocation($a_type);
 		$full_class = "ilObj".$class."Access";
-		include_once($location."/class.".$full_class.".php");
```

Comment:
@rsheer I forgot this: How about plugins (RepoObj-Plugins), do they still work with this removal?

Reply:
@chfsx good point. I tested it with a generic RepoObj-Plugin and with LiveVoting. LiveVoting worked (I think because of its own autoloader). But the generic plugin got a "class not found"-exception in ilObjectFactory:

        $obj = new $class_name(0, false);    // this avoids reading of data

So I think the includes are still needed.

Reply:
Thanks for checking this... yes please re-add the line again